src/Tribe/Utils/Taxonomy.php

```diff
@@ -108,7 +108,7 @@ public static function normalize_to_term_ids( $terms, $taxonomy ) {
 
 		$terms = array_map( static function ( $param ) use ( $taxonomy ) {
 			$param   = preg_replace( '/^#/', '', $param );
-			$term_by = is_numeric( $param ) ? 'ID' : 'slug';
+			$term_by = is_int( $param ) ? 'ID' : 'slug';
```

Comment:
`preg_replace()` would return a string in this case, would it not? Therefore `is_int()` would never be true. 

I think it would be better, if `$param` is numeric, to check by `ID` and see if a term is returned. If not, check by `slug`.